runelite-client/src/test/java/net/runelite/client/plugins/discord/DiscordStateTest.java

```diff
@@ -83,7 +81,8 @@
 	public void before()
 	{
 		Guice.createInjector(BoundFieldModule.of(this)).injectMembers(this);
-		when(partyService.getLocalPartyId()).thenReturn(UUID.nameUUIDFromBytes("test".getBytes(StandardCharsets.UTF_8)));
+		// mockito UnnecessaryStubbingException
```

Comment:
We use git, so you can just remove lines of code instead commenting them out, and still see them in the git history if you need them again.